pkg/healthmonitor/metrics.go

```diff
@@ -0,0 +1,64 @@
+package healthmonitor
+
+import (
+	compbasemetrics "k8s.io/component-base/metrics"
+)
+
+type registerables []compbasemetrics.Registerable
+
+var (
+	healthyTargetsTotal = compbasemetrics.NewCounterVec(
+		&compbasemetrics.CounterOpts{
+			Name:           "health_monitor_healthy_target_total",
+			Help:           "Number of healthy instances registered with the health monitor. Partitioned by targets",
+			StabilityLevel: compbasemetrics.ALPHA,
+		},
+		[]string{"target"},
+	)
+
+	unHealthyTargetsTotal = compbasemetrics.NewCounterVec(
+		&compbasemetrics.CounterOpts{
+			Name:           "health_monitor_unhealthy_target_total",
+			Help:           "Number of unhealthy instances registered with the health monitor. Partitioned by targets",
+			StabilityLevel: compbasemetrics.ALPHA,
+		},
+		[]string{"target"},
+	)
```

Comment:
we need metrics for:
- counter for toggling
- counter for network issues (connection refused) that are not following a period of N seconds of red `/readyz`

Reply:
> counter for toggling

I think that having the total for healthy/unhealthy is enough to alert on `toggling`

Reply:
> counter for network issues (connection refused) that are not following a period of N seconds of red /readyz

do you mean detect connection refused errors when an ep turned red on `/readyz` since X seconds ago?

Reply:
> do you mean detect connection refused errors when an ep turned red on /readyz since X seconds ago?

Want to count when endpoint become unhealthy without following the `/readyz` protocol.